examples/example_wallet_esplora_async/Cargo.toml

```diff
@@ -8,5 +8,5 @@ edition = "2021"
 [dependencies]
 bdk_wallet = { path = "../../wallet", features = ["rusqlite"] }
 bdk_esplora = { version = "0.20", features = ["async-https", "tokio"] }
-tokio = { version = "1", features = ["rt", "rt-multi-thread", "macros"] }
+tokio = { version = "1.44.2", features = ["rt", "rt-multi-thread", "macros"] }
```

Comment:
The vulnerability looks like it was patched for versions >=1.23.1, so fine to set this to our MSRV version and let `cargo update` do it's thing to bump it when using `stable`.
```suggestion
tokio = { version = "1.38.1", features = ["rt", "rt-multi-thread", "macros"] }
```